HandsOnTutorial/0-provision-azure-cluster.fsx

```diff
@@ -1,168 +1,72 @@
-to [sign up](https://azure.microsoft.com/en-us/pricing/free-trial/) for an Azure subscription. Once signed up, follow 
-the [instructions](https://msdn.microsoft.com/en-us/library/dn414650(v=nav.90).aspx) on how to download your private
-`.publishsettings` file, which contains all authentication information necessary to manage your Azure subscription(s).
-
-*)
-
-let pubSettingsFile = @"C:\path\to\my.publishsettings" // replace with your local .publishsettings path
-
-(**
-
-Next, read your publish settings file by calling
-
-*)
-
-let pubSettings = PublishSettings.Parse pubSettingsFile
-
-(** 
-
-Yielding
-
-    val pubSettings : PublishSettings =
-      [|"Visual Studio Premium with MSDN"; "Azure in Open"; "Azure Free Trial"|]
-
-Now we can select our subscription of preference
-
-*)
-
-let subscription = pubSettings.GetSubscriptionById "Azure Free Trial"
-
-(**
-
-Select your prefered Azure region. This specifies the default Azure data center to be used for your deployments.
-Choosing a data center close to your location is recommended.
-
-*)
-
-let myRegion = Region.North_Europe
-
-(**
-
-Let's now instantiate our subscription manager instance
-
-*)
-
-let manager = SubscriptionManager.Create(subscription, myRegion)
-
-(**
-
-Provisioning an MBrace cluster is now straightforward. Simply write
-
-*)
-
-let deployment = manager.Provision(serviceName = "mbracetest", vmCount = 4, vmSize = VMSize.A3)
-
-(**
-
-Which should provision a 4-worker MBrace cluster of Large (A3) instances
-Provisioning can take some time (approximately 5 minutes). We can track progress by running
+to provision an MBrace cluster using F# Interactive. In order to proceed, you will need
+to [sign up](https://azure.microsoft.com/en-us/pricing/free-trial/) for an Azure subscription. Once signed up, 
+[download your publication settings file ](https://manage.windowsazure.com/publishsettings).
 
 *)
 
-deployment.ShowInfo()
+// replace with your local .publishsettings path
+let pubSettingsFile = @"C:\path\to\my.publishsettings" 
 
 (**
 
-Which yields
-
-    Cloud Service "mbracetest"
-
-    Name        Region        VM size  #Instances  Service Status  Deployment Status   Last Modified
-    ----        ------        -------  ----------  --------------  -----------------   -------------
-    mbracetest  North Europe  Large    4           Created         Provisioning 33.3%  19/11/2015 6:24:01 pm
-
-Finally, when deployment has completed we can easily obtain a cluster instance by typing
+Next, specify your region and the size and number of your virtual machines, and create your cluster:
 
 *)
-
-let cluster = AzureCluster.Connect deployment
+    
+let region = Region.North_Europe
+let vmSize = VMSize.Large
+let vmCount = 4
+    
+let deployment = SubscriptionManager.Provision(pubSettingsFile, region, vmSize=vmSize, vmCount=vmCount) 
 
 (**
-
-and submit our first computation to the cluster
+This will provision a 4-worker MBrace cluster of Large (A3) instances
```

Comment:
I think it is important to add a deployment.ShowInfo() here. Omitting this might leave new users in the dark as regards provisioning progress. Also, directly initializing the cluster object below will result in an empty worker set, which may confuse users.